packs/equipment-effects/effect-deathdrinking-living.json

```diff
@@ -0,0 +1,47 @@
+{
+    "_id": "VIYfrecOd54NUPur",
+    "img": "systems/pf2e/icons/equipment/runes/weapon-property-runes/weapon-property-runes.webp",
+    "name": "Effect: Deathdrinking (Living)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Deathdrinking]</p>\n<p>You gain a +1 item bonus to attack and damage rolls for 10 minutes.</p>"
```

Comment:
Since these have the same duration we would want a single effect with a ChoiceSet to decide between the two modes, rather than using two effects.